internal/provider/provider.go

```diff
@@ -312,7 +337,7 @@ func (p *pingfederateProvider) Configure(ctx context.Context, req provider.Confi
 	clientConfig.DefaultHeader["X-BypassExternalValidation"] = strconv.FormatBool(xBypassExternalValidation)
 	clientConfig.Servers = client.ServerConfigurations{
 		{
-			URL: httpsHost + "/pf-admin-api/v1",
+			URL: httpsHost + adminApiPath,
```

Comment:
Do we need to have these as separate variables? Would it make more sense to just have one entire host variable including the admin api path?

Reply:
This is a breaking change for current users - I've asked Wes to revert

Reply:
fixed and merged https://github.com/pingidentity/terraform-provider-pingfederate/pull/170